Tribler/Test/Community/Multichain/StatisticsDispaly/test_statistics_database.py

```diff
@@ -0,0 +1,73 @@
+from Tribler.Test.test_as_server import BaseTestCase
+from Tribler.community.multichain.StatisticsDisplay.networkNode import NetworkNode
+from Tribler.community.multichain.StatisticsDisplay.dbDriver import DbDriver
+
+from twisted.internet.defer import inlineCallbacks
```

Comment:
Change the order of imports; first all imports from the Python standard library, followed by imports from third-party dependencies, followed by imports from Tribler. Within these groups, alphabetically sorted.